go/testskeleton/testskeleton.go

```diff
@@ -0,0 +1,354 @@
+package testskeleton
```

Comment:
As @pimielowski suggested, I propose to remove folder `go` with all `*.go` files and use external packages from https://github.com/PaloAltoNetworks/terraform-modules-vmseries-tests-skeleton.

Reply:
👍🏻 unless the code required by Azure did not get released yet

Reply:
yes, but [PR 3](https://github.com/PaloAltoNetworks/terraform-modules-vmseries-tests-skeleton/pull/3) is approved, we can merge it and execute manually release action (to not to wait until Thursday) :)

Reply:
fixed